Gemfile

```diff
@@ -1,5 +1,5 @@
 raise "Ruby versions < 3.0.1 are unsupported!"  if RUBY_VERSION < "3.0.1"
-raise "Ruby versions >= 3.2.0 are unsupported!" if RUBY_VERSION >= "3.2.0"
+raise "Ruby versions >= 3.4.0 are unsupported!" if RUBY_VERSION >= "3.4.0"
```

Comment:
As discussed, let's just add a warn here that 3.2+ are not fully tested yet

Reply:
But it can be done in a follow up.

Reply:
done